### PR TITLE
fix: THEEDGE-3551 issue in system update/cancel

### PR DESCRIPTION
**Pull request by mgold1234**
fixes: #THEEDGE-3551

# Description
this commit fix the problem when user try to press on Update system/cancel button on system details screen

https://github.com/RedHatInsights/edge-frontend/assets/73419853/81018960-41f1-4540-a45d-e0442a66264c

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: src/Routes/Devices/UpdateSystem.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ const UpdateSystemMain = ({
     const pathLen = destPath.split('/').length;
     const dest = pathname.split('/').slice(0, pathLen).join('/');
     if (navigateProp) {
-      navigate({ ...dest });
+      navigate(paths.insightsInventory);
```
Comment (ldjebran) [inline src/Routes/Devices/UpdateSystem.js:214]: think this will not work in system detail view

Comment (mgold1234) [reply src/Routes/Devices/UpdateSystem.js:235]: its working in both system

**Review by ldjebran — APPROVED** [review]
lgtm,
note that insight/inventory  system details view is missing update button (that is not related to this bug fixing PR)

**Review by adeliaferreira — APPROVED** [review]
Lgtm

**Pull request merged**